packages/rocketchat-lib/server/models/Messages.js

```diff
@@ -318,10 +320,34 @@ RocketChat.models.Messages = new class extends RocketChat.models._Base {
 		return this.insert(record);
 	}
 
+	findJoinRequestsByRoom(msgType, rid) {
```

Comment:
Also here: Name does not match signature.

Reply:
renamed as `findJoinRequestByMsgTypeAndRoom`